src/main/java/org/eclipse/uprotocol/client/RpcMapper.java

```diff
@@ -47,18 +36,18 @@ public interface RpcMapper {
      * @return Returns a CompletionStage containing the declared expected return type of the RPC method or an exception.
```

Comment:
some docstrings in this file need to be updated with UPayload from UMessage, see line 33

Reply:
Did more documentation cleanup